lib/src/Base/Func/EvaluationImplementation.cxx

```diff
@@ -555,6 +555,119 @@ Graph EvaluationImplementation::draw(const Point & xMin,
   return draw(0, 1, 0, Point(2), xMin, xMax, pointNumber, scale);
 }
 
+/** Draw the cross-cuts of the function supposed to have 1D output */
+GridLayout EvaluationImplementation::drawCrossCuts(const Point& centralPoint,
+  const Point& xMin,
+  const Point& xMax,
+  const Indices& pointNumber,
+  const Scalar vMin,
+  const Scalar vMax,
+  const Bool withMonoDimCuts,
+  const Bool isFilled) const
+{
+  UnsignedInteger inputDim = getInputDimension();
+  if(!(getInputDimension() >= 2)) throw InvalidArgumentException(HERE) << "Error: cannot draw cross cuts of a function with input dimension=" << getInputDimension() << " less than 2 using this method. See the other draw() methods.";
+  if(getOutputDimension() != 1) throw InvalidArgumentException(HERE) << "Error: cannot draw cross cuts of a function with output dimension=" << getOutputDimension() << " different from 1 using this method. See the other draw() methods.";
+  if(!(xMin.getDimension() == inputDim && xMax.getDimension() == inputDim && pointNumber.getSize() == inputDim)) throw InvalidArgumentException(HERE) << "Error: xMin, xMax and PointNumber must be of dimension " << inputDim;
+  for(UnsignedInteger i=0;i<inputDim;i++)
+    if(!(pointNumber[i] > 2)) throw InvalidArgumentException(HERE) << "Error: the discretization must have at least 2 points per component";
+  Bool buildVMinMax = vMax == -HUGE_VAL && vMin == -HUGE_VAL;
+  Scalar vMinCal = buildVMinMax ? HUGE_VAL : vMin, vMaxCal = vMax;
+  if(!buildVMinMax && !(vMin < vMax))throw InvalidArgumentException(HERE) << "Error: the vMin value must be less than the vMax value";
+  //Building component samples
+  std::vector<Sample>samples(inputDim);
+  for(UnsignedInteger iX = 0; iX < inputDim; iX++)
+  {
+    UnsignedInteger nX = pointNumber[iX];
+    SampleImplementation sample(nX, 1);
+    const Scalar dX = (xMax[iX] - xMin[iX]) / (nX - 1.0);
```

Comment:
```suggestion
  const Scalar dX = deltaX[iX];
```

Reply:
`deltaX` is not defined, what do you mean?

Reply:
?